Update README.md (#38)
use-inkubator/Ecosystem-Grants: README.md

- [\:pencil: Process](#pencil-process)
  - [1. Application](#1-application)
  - [2. Application Review](#2-application-review)
  - [3. Milestone Delivery and Payment](#3-milestone-delivery-and-payment)
  - [Changes to a Grant after Approval](#changes-to-a-grant-after-approval)
- [\:mailbox_with_mail: Suggest a Project](#mailbox_with_mail-suggest-a-project)
- [\:bulb: Help](#bulb-help)
  - [Real-time conversation](#real-time-conversation)
- [FAQ](https://github.com/use-inkubator/Ecosystem-Grants/blob/master/FAQ.md)

<img src="./assets/inkubator-tanks.svg" style="width: 100%;">

**👋 Introduction**

Ecosystem Grants are part of the **ink!ubator**, a holistic bounty program with the goal of furthering the ink! smart contracting ecosystem. This bounty program is funded by the Polkadot treasury with the support of the community.

The bounty includes Curators from the following Polkadot ecosystem teams:

- [W3F](https://web3.foundation/)
- [Michi](https://github.com/cmichi)
- [Velocity Labs](https://www.velocitylabs.org/)
- [David Semakula](https://davidsemakula.com/)
- [R0GUE](https://r0gue.io/)
- **Facilitator**:
    - [Toma Sadova](https://www.linkedin.com/in/toma-sadova/)

> The Ecosystem Grants are only one aspect of the ink!ubator. For more details on the rest of the initiative, see the [Polkadot Treasury Bounty Proposal](https://polkadot.polkassembly.io/bounty/19).

Reviewer: ```suggestion
```

We take licensing and proper attribution very seriously. Any misuse of others' work without proper credit will lead to immediate termination of the grant. If in doubt, reach out before submitting.

---

**Project Ideas**

The main acceptance criteria for any submitted project is its impact on the ink! ecosystem. Suggested categories include:

### Business Dapp

This category is for early-stage startups utilizing ink! that seek grant support. Applicants must demonstrate a viable business model, a capable team, and readiness to transform their idea into a functional company. A pitch deck is encouraged.

### Canary Dapp

A Canary Dapp is a full-stack application that **could** be deployed on a value-bearing network but may not be fully production-ready. These projects serve as references for developers building production-grade dApps.

Examples:

- **Lending application** for earning interest on deposits and borrowing assets.
- **Liquid staking** solutions issuing redeemable receipts for staked tokens.
- **Aggregators** uniting decentralized protocols and liquidity sources.

> We recommend using [Sol2ink](https://github.com/Brushfam/sol2ink) to accelerate development. Learn more in the [Sol2ink docs](https://learn.brushfam.io/docs/sol2ink).

Reviewer: ```suggestion
```

- [Project Ideas](#project-ideas)
- [Bounty Levels](#bounty-levels)
- [Support](#support)
- [Team](#team)
- [\:pencil: Process](#pencil-process)
  - [1. Application](#1-application)
  - [2. Application Review](#2-application-review)
  - [3. Milestone Delivery and Payment](#3-milestone-delivery-and-payment)
  - [Changes to a Grant after Approval](#changes-to-a-grant-after-approval)
- [\:mailbox_with_mail: Suggest a Project](#mailbox_with_mail-suggest-a-project)
- [\:bulb: Help](#bulb-help)
  - [Real-time conversation](#real-time-conversation)
- [FAQ](https://github.com/use-inkubator/Ecosystem-Grants/blob/master/FAQ.md)

<img src="./assets/inkubator-tanks.svg" style="width: 100%;">

**👋 Introduction**

Ecosystem Grants are part of the **ink!ubator**, a holistic bounty program with the goal of furthering the ink! smart contracting ecosystem. This bounty program is funded by the Polkadot treasury with the support of the community.

The bounty includes Curators from the following Polkadot ecosystem teams:

- [W3F](https://web3.foundation/)
- [Michi](https://github.com/cmichi)
- [Velocity Labs](https://www.velocitylabs.org/)
- [David Semakula](https://davidsemakula.com/)

> - [R0GUE](https://github.com/al3mart)

Reviewer: ```suggestion
- [R0GUE](https://r0gue.io/)
```

- **Facilitator**:
    - [Toma Sadova](https://www.linkedin.com/in/toma-sadova/)

> The Ecosystem Grants are only one aspect of the ink!ubator. For more details on the rest of the initiative, see the [Polkadot Treasury Bounty Proposal](https://polkadot.polkassembly.io/bounty/19).

Reviewer 2: Is this still true?

Author: agree, we can remove it

> The Ecosystem Grants are only one aspect of the ink!ubator. For more details on the rest of the initiative, see the [Polkadot Treasury Bounty Proposal](https://polkadot.polkassembly.io/bounty/19).

Reviewer: Since the contents of the  bounty page are out of date (with regards to ink!ubator 2.0), I think we should replace this link with one for the "new proposal" i.e. one of the following
- https://forum.polkadot.network/t/ink-ubator-2-0-proposal-for-treasury-funding-to-accelerate-polkadots-growth-through-ink-smart-contracts/10537
- https://polkadot.polkassembly.io/referenda/1331

Reviewer: Resolving this in favor of removing this section entirely

The bounty includes Curators from the following Polkadot ecosystem teams:
- [Astar Network](https://astar.network/)
- [Parity](https://www.parity.io/)
- [Phala Network](https://www.phala.network/en/)
- [Brushfam](https://brushfam.io/)
- [Aleph Zero](https://alephzero.org/)

The Ecosystem Grants focus on funding projects which will demonstrate the potential of ink! and well as those which push the ink! ecosystem forward, such as infrastructure projects or developer tooling.
- [W3F](https://web3.foundation/)
- [Michi](https://github.com/cmichi)
- [Velocity Labs](https://www.velocitylabs.org/)
- [David Semakula](https://davidsemakula.com/)
- [R0GUE](https://github.com/al3mart)
- **Facilitator**:
    - [Toma Sadova](https://www.linkedin.com/in/toma-sadova/)

The Ecosystem Grants are only one aspect of the ink!ubator. For more details on the rest of the initiative, see the [Polkadot Treasury Bounty Proposal](https://polkadot.polkassembly.io/bounty/19).

---

The Ecosystem Grants are only one aspect of the ink!ubator. For more details on the rest of the initiative see the [Polkadot Treasury Bounty Proposal](https://polkadot.polkassembly.io/bounty/19).
**Guidelines**

> ### Guidelines

Reviewer 2: If you remove the headings ("\#\#\#"), the links in the table of contents won't work anymore.

We take licensing and proper attribution very seriously. Any misuse of others' work without proper credit will lead to immediate termination of the grant. If in doubt, reach out before submitting.

---

> **Project Ideas**

Reviewer 2: see above

The main acceptance criteria for any submitted project is its impact on the ink! ecosystem. Suggested categories include:

### Business Dapp

This category is for early-stage startups utilizing ink! that seek grant support. Applicants must demonstrate a viable business model, a capable team, and readiness to transform their idea into a functional company. A pitch deck is encouraged.

### Canary Dapp

A Canary Dapp is a full-stack application that **could** be deployed on a value-bearing network but may not be fully production-ready. These projects serve as references for developers building production-grade dApps.

Examples:

- **Lending application** for earning interest on deposits and borrowing assets.
- **Liquid staking** solutions issuing redeemable receipts for staked tokens.
- **Aggregators** uniting decentralized protocols and liquidity sources.

> We recommend using [Sol2ink](https://github.com/Brushfam/sol2ink) to accelerate development. Learn more in the [Sol2ink docs](https://learn.brushfam.io/docs/sol2ink).

Reviewer 2: This repo is not maintained. Do we have a more recent version or an alternative?

Author: I don’t know the alternatives

### Technical Showcase

This category involves applications demonstrating ink!'s advantages over other smart contract languages, such as:

- Efficient gas usage
- Decreased contract size
- Improved developer and debugging experience

### Infrastructure

This category focuses on improving the ink! ecosystem's developer infrastructure and tooling. Innovative ideas that add differentiation to Polkadot/ink! are preferred over duplications of EVM tools.

Examples of successful infrastructure products:

- [OpenBrush](https://openbrush.io/)
- [Swanky](https://github.com/AstarNetwork/swanky-cli)
- Oracles (price feeds, VRF, customizable oracles, etc.)
- DAO tooling (smart contracts, SDKs, UIs)
- Bridges (XCM integration, external blockchain bridges)

---

**Bounty Levels**

The Smart Contract Bounty Program offers different levels based on your stage:

### 🐣 Level 1

- **Target:** Individuals & small teams
- **Amount:** Up to $10,000
- **Requirements:** 4 approvals

### 🐤 Level 2

- **Target:** Small teams/startups
- **Amount:** Up to $30,000
- **Requirements:** 4 approvals

### 🐓 Level 3

- **Target:** Companies/foundations with proven track records
- **Amount:** Up to $250,000

> - **Requirements:** 5 approvals (for >$100k: Smart Contract Bounty Council approval + optional pitch call)

Reviewer 2: Who is the Smart Contract Bounty Council?

Author: The curators are

---

**Benefits**

- Feedback during application and evaluation
- Introductions to relevant teams/projects
- Code reviews or audits (depending on scope)
- Co-promotion opportunities

---

**Support**

While funding and milestone feedback are the main support mechanisms, general documentation and additional resources can be found at [use.ink](https://use.ink/).

Community support channels:

> - [Smart contracts & Parity's ink! Element](https://matrix.to/#/#ink:matrix.parity.io)

Reviewer 2: This room is closed. Maybe https://t.me/inkathon instead?

- [Brushfam ink! community Discord](https://discord.gg/9FRETSPmp9)

---

> **📝 Process**

Reviewer 2: See above

> 📢 Payment is made in DOT (on Polkadot).

### 1. Application

> 1. [Fork](https://github.com/smart-contract-bounty/Wasm-Bounty-01/fork) this repository.
> 2. Create a copy of the application template ([`applications/application-template.md`](https://github.com/smart-contract-bounty/Ecosystem-Grants/blob/master/applications/application-template.md)).

Reviewer 2: ```suggestion
1. [Fork](https://github.com/use-inkubator/Ecosystem-Grants/fork) this repository.
2. Create a copy of the application template ([`applications/application-template.md`](https://github.com/use-inkubator/Ecosystem-Grants/blob/master/applications/application-template.md)).
```

3. Name the new file after your project: `project_name.md`.
4. Complete the template with as much detail as possible.
5. Submit a pull request to this repository. Only include **one new file** in your pull request.

### 2. Application Review

1. Curators will review and comment on your pull request.
2. Address feedback by modifying your application.
3. Once ready, the application will be marked as `ready for review` and shared with the committee.
4. Approval requires **4 votes** and will mark the project start date.

### 3. Milestone Delivery and Payment

> Milestones are to be delivered in the [Grant Milestone Delivery](https://github.com/smart-contract-bounty/Grant-Milestone-Delivery) repository.

Reviewer 2: ```suggestion
Milestones are to be delivered in the [Grant Milestone Delivery](https://github.com/use-inkubator/Grant-Milestone-Delivery) repository.
```

---

> **💡 Help**

Reviewer 2: see above

For real-time support, join:

- [Ecosystem Grants Community](https://matrix.to/#/#sm-bounty-support:matrix.org)

Additional channels:

> - [Smart contracts & Parity's ink!](https://matrix.to/#/#ink:matrix.parity.io)

Reviewer 2: see above

> - [Brushfam ink! community](https://discord.gg/9FRETSPmp9)

Reviewer 2: This looks abandoned.

Reviewer 3: Indeed.
https://x.com/0xMarkian/status/1702384661605867759

Reviewer 3: BTW, Pendzl is successor of the Openbrush library. It is audited and compatible with ink! 5.0.0 and was granted by W3F.

https://github.com/Pendzl/pendzl
